source/js/main.js

```diff
@@ -26,6 +26,49 @@ let main = {
         this.reactPrimaryNav.hide();
       }
     });
+
+    // Track window scroll position and add/remove class to change sticky header appearance
+
+    let lastKnownScrollPosition = 0;
+    let ticking = false;
+    let elBurgerWrapper = document.querySelector(`.wrapper-burger`);
+
+    let adjustNavbar = (scrollPosition) => {
+      if (scrollPosition > 100) {
```

Comment:
Is there a way to not use hardcoded number here? If not, can we put a comment here explaining where the hardcoded number `100` came from so that number doesn't look so magical?